components/admin_console/permission_schemes_settings/permission_system_scheme_settings/permission_system_scheme_settings.test.jsx

```diff
@@ -168,8 +168,6 @@ describe('components/admin_console/permission_schemes_settings/permission_system
         expect(wrapper.state().showResetDefaultModal).toBe(false);
         wrapper.find('.reset-defaults-btn').first().simulate('click');
         expect(wrapper.state().showResetDefaultModal).toBe(true);
-        wrapper.find('.btn-cancel').first().simulate('click');
```

Comment:
I would expect to have this checks still there

Reply:
Because it's a shallow render it is not longer available to this test.

Reply:
Note the shallow render no longer allows access to the confirm modal confirm button.